packages/view-types/feature-list/src/selectable-table/SelectableTable.js

```diff
@@ -35,6 +35,7 @@ export default function SelectableTable(props) {
   const {
     hasColorEncoding,
     columns,
+    columnLabels,
```

Comment:
No need to make it an object, but please add a JSDoc comment for `columnLabels` that mentions it is expected to be the same length as `columns`

Reply:
Makes sense, I have done that in this commit: 28eb48df404d85d7d94a17d875550e92161c37db Feel free to resolve this comment if it looks ok.